announcing-community-cdn.rst

```diff
@@ -0,0 +1,80 @@
+Shipping a CDN on Read the Docs Community
+=========================================
+
+You might have noticed that our Read the Docs Community site has gotten faster in the past few weeks.
+How much faster likely depends on how far away you live from Virginia,
+which is where our servers have traditionally lived.
+
+We have recently enabled a CDN on **all Read the Docs Community sites**,
+generously sponsored by `CloudFlare`_.
+This post will talk a bit more about how we implemented this,
+and why we're excited about it.
+
+Hosting thousands of domains is hard
+------------------------------------
+
+Traditionally the largest problem that we've had with rolling out features to all of our documentation sites is scale.
+We host thousands of custom domains for our users,
+and any solution needs to work across all of them.
+This has presented a number of issues in the past,
+but CDN is one of the most complicated.
+
+To make a CDN function across all our sites,
+every data center that the CDN has needs to work with all our custom domains.
+Imagine we have 5,000 domains and there are 100 data centers across the world,
+this means that ``5,000 * 100 = 500,000`` different endpoints need to be configured for this to work at scale.
+
+We are lucky that CloudFlare has the global scale to be able to donate us this service.
+Specifically,
+their `SSL for SAAS`_ service is what we're using for both SSL and CDN across all our custom domains
+This allows us to offload the complexity to CloudFlare,
+and only focus on our integration.
+
+Implementation
+--------------
+
+One of the coolest things that CloudFlare's CDN offers is something called `Cache Tags`_.
+This lets us add a ``Cache-Tag`` header to all the documentation that we serve,
+and bust the cache using just that tag.
+
+An example,
+when you load our docs,
+we will return a header::
+
+    GET https://docs.readthedocs.io/en/latest/
+    ...
+    Cache_tag: docs-latest,docs
+
+We return a tag that matches both the ``$project``, and the ``$project-$version``.
+This allows us to bust the cache for a specific version,
```

Comment:
```suggestion
This allows us to invalidate the cache for a specific version,
```

Invalidate sounds more familiar to me when talking about cache.

Reply:
Hrm... There are more uses of the "bust" word in the next paragraphs. I assume it's just to not repeat yourself too much.